bangazonapi/views/profile.py

```diff
@@ -7,7 +7,7 @@
 from rest_framework.permissions import IsAuthenticatedOrReadOnly
 from rest_framework.response import Response
 from rest_framework.viewsets import ViewSet
-from bangazonapi.models import Order, Customer, Product, OrderProduct, Favorite
+from bangazonapi.models import Order, Customer, Product, OrderProduct, Favorite, payment
```

Comment:
why is `payment` here?

Reply:
Hmmmm not sure! Must have typed it there thinking the cursor was somewhere else. I removed it just now and everything is still functional. pushing the changes now